### DISCUSSION TITLE
Number of selected events doesn't match the number of generated events

### DISCUSSION THREAD

**Discussion by mmarko5** [MadAnalysis/madanalysis5#199] [Q&A]
Dear experts,

I generate the BSM process in Madgraphp p > t t~ vt vt~ QCD==2 NP==2, with a leptoquark as the BSM particle. I request 200k events, and the run goes smoothly. I then plot for example the missing transverse energy in MadAnalysis5. Here comes the problem:
when I impose some selection cuts, I can see the number of accepted and rejected events. However, those numbers are always extremely small compared to the total number of events. For example, if I require: ma5> select MET < 5000, I would expect 200k accepted events, but I only have 0.217 (and 0.0 rejected events). I don't know whether I misunderstood something in how the program works, or if there really is a problem. I would appreciate any help !

Thank you in advance,
Marko

**Comment by BFuks** [comment]
Please see [here](https://github.com/MadAnalysis/madanalysis5/issues/198). There is no need to post this twice.
